### PR TITLE
Generate a PDF with Chrome using the Knit button of RStudio IDE

### PR DESCRIPTION
**Pull request by RLesur**
Here's a proposal to create a custom render function for the RStudio IDE.  
My main problem is to give a name for this new function :smiley: (that's why I open a PR).

In short, by adding `knit: pagedown::knit_pdf_chrome` in the YAML header, a PDF is also generated by clicking on the "Knit" button of RStudio.  
I don't like very much the current name `knit_pdf_chrome` but I have no better idea. I'd appreciate any proposal.

This PR also enhances the documentation about printing to PDF.

### CODE REVIEW

**Comment by RLesur** [comment]
> Does this proposal sound good?

Of course... because I had already thought about that :smile: 
I like this idea very much. I will try.

**Comment by RLesur** [comment]
> Then we call `rmarkdown::render(..., quiet = is_rstudio_knit)`

I already tried this. In this case, this also removes the **knitr** progress messages because the `quiet` argument is passed to `knitr::knit()` by `rmarkdown::render()`.  
This leads to a different user experience. That's why I used `suppressMessages()`.

**Comment by yihui** [comment]
I wonder what happens if you don't change the `quiet` argument, but just add another `message()` in the end. When there are two messages, RStudio might be open the file in the last message. You can test it.

**Comment by RLesur** [comment]
> You can test it.

I've already tested it :wink: and RStudio IDE opens the file in the first message.

File: inst/examples/index.Rmd [hunk 1]
```diff
@@ -84,8 +84,51 @@ Note that this overriding mechanism also works for other output formats in **pag
...
+   ```yaml
+   knit: pagedown::chrome_print
+   ```
+   
+   With this metadata parameter, the behavior of the "Knit" button of RStudio is modified: it produces both the HTML document and the PDF with Chrome. This functionality is suitable for any R Markdown HTML output format and is mainly convenient for small documents like presentations or notes.  
+   If `chrome_print()` cannot find Chrome or Chromium automatically, set the `PAGEDOWN_CHROME` environment variable to the path to Chrome or Chromium executable. 
+
+### Print to PDF on a server
+
+If you want to use `chrome_print()` or `knit_pdf_chrome()` on a server (an RStudio server, for instance), Chromium or Chrome has to be present on this server and available from the `PATH` or `PAGEDOWN_CHROME` environment variables.
```
Comment (cderv) [inline inst/examples/index.Rmd]: `knit_pdf_chrome` is no more existing right ? 
I think it is a leftover from your first propsal.

Comment (RLesur) [reply inst/examples/index.Rmd]: Good catch! 🕵

Comment (RLesur) [reply inst/examples/index.Rmd]:
```suggestion
If you want to use `chrome_print()` on a server (an RStudio server, for instance), Chromium or Chrome has to be present on this server and available from the `PATH` or `PAGEDOWN_CHROME` environment variables.  
```

**Pull request merged**